ibmdbpy/frame.py

```diff
@@ -227,6 +227,15 @@ def indexer(self):
         else:
             None
 
+    #### added ####
```

Comment:
This comment is not useful - of course someone added this function at some point, otherwise it wouldn't exist ;-) Please remove it.